lib/pure/asyncmacro.nim

```diff
@@ -348,7 +348,7 @@ proc asyncSingleProc(prc: NimNode): NimNode {.compileTime.} =
     newVarStmt(retFutureSym,
       newCall(
         newNimNode(nnkBracketExpr, prc.body).add(
-          newIdentNode(!"newFuture"), # TODO: Strange bug here? Remove the `!`.
+          newIdentNode("newFuture"), # TODO: Strange bug here? Remove the `!`.
```

Comment:
The TODO comment is outdated now.

Reply:
Well it seems I just didn't see it. I used a simple search and replace here.